frontend/package.json

```diff
@@ -6,13 +6,29 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
+    "algoliasearch": "^3.35.1",
+    "feather-icons": "^4.28.0",
+    "framer-motion": "^2.9.4",
+    "instantsearch.css": "^7.4.5",
     "react": "^16.14.0",
+    "react-anchor-link-smooth-scroll": "^1.0.12",
     "react-dom": "^16.14.0",
-    "react-scripts": "3.4.3"
+    "react-instantsearch-dom": "^6.8.2",
+    "react-modal": "^3.11.2",
+    "react-rnd": "^10.2.3",
+    "react-router-dom": "^5.2.0",
+    "react-scripts": "^4.0.0",
+    "react-slick": "^0.27.12",
+    "slick-carousel": "^1.8.1",
+    "styled-components": "^5.2.0",
+    "tailwindcss": "^1.9.6",
+    "twin.macro": "^1.11.0",
+    "use-in-view": "^1.0.15"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "BROWSER='firefox developer edition' react-scripts start",
     "build": "react-scripts build",
+    "buildtailwind": "postcss src/styles/tailwind.css -o public/build/tailwind.css",
```

Comment:
If this script has to be called before/after main build process, I'd prefer you to include it into the build script itself — for the sake of automated build. Otherwise we'll have to adjust the DevOps workflow, which is possible but not desirable.

Reply:
OK this script is removed.